homeassistant/components/daikin/const.py

```diff
@@ -4,11 +4,16 @@
 ATTR_TARGET_TEMPERATURE = "target_temperature"
```

Comment:
This should be merged with the import above..
```suggestion
from homeassistant.const import CONF_DEVICE_CLASS, DEVICE_CLASS_POWER, DEVICE_CLASS_TEMPERATURE


ATTR_TARGET_TEMPERATURE = "target_temperature"
```